dashboard/app/models/census/census_summary.rb

```diff
@@ -168,18 +295,43 @@ def self.summarize_census_data
             school_year: school_year,
           )
 
-          summary.audit_data = JSON.generate(audit)
+          # Assuming some chance that the school will stop teaching we degrade the
+          # old belief.
+          # Use an uninformed prior (0.5) if we don't have a previous year's belief.
+          prior = previous_years_belief ? (previous_years_belief * 0.9) : 0.5
```

Comment:
I'm not sure what's standard for this type of model, but given that (I assume) the base rate of teaching CS is low as a percent of all schools, is .5 a reasonable uninformed prior to use? (Seems high)

Reply:
0.5 is the *uninformed* prior because it doesn't take any outside information into account and just says that without any data we have no preference for thinking that the school does or does not teach CS. A more informed value could be something around 0.4 since that is the number we are currently reporting. In practice that doesn't make much difference to the posteriors once we have some data about a school.